tests/test_schemas.py

```diff
@@ -40,21 +40,55 @@ def generate_graph_data_model(helpers, path_to_data_model):
 
 @pytest.fixture
 def DME(helpers, data_model_name='example.model.csv'):
-    path_to_data_model = helpers.get_data_path("example.model.jsonld")
-
-    graph_data_model = generate_graph_data_model(helpers, path_to_data_model=path_to_data_model)
+    '''
+    In future could pull using helpers.
+    '''
+    graph_data_model = generate_graph_data_model(helpers, data_model_name=data_model_name)
     DME = DataModelGraphExplorer(graph_data_model)
     yield DME
 
+
 class TestDataModelParser:
     def test_get_base_schema_path(self, helpers):
-        return
-
-    def test_get_model_type(self):
-        return
-
-    def test_parse_model(self):
-        return
+        '''Test that base schema path is returned properly.
+        Note:
+            data model parser class does not currently accept an new path to a base schema,
+            so just test that default BioThings data model path is returned.
+        '''
+        # Instantiate Data model parser.
+        data_model_parser = helpers.get_data_model_parser(data_model_name='example.model.csv')
+
+        # Get path to default biothings model.
+        biothings_path = data_model_parser._get_base_schema_path(base_schema=None)
+
+        assert os.path.basename(biothings_path) == "biothings.model.jsonld"
+
+    @pytest.mark.parametrize("data_model", ['example.model.csv', 'example.model.jsonld'], ids=["csv", "jsonld"])
+    def test_get_model_type(self, helpers, data_model):
+        # Instantiate Data model parser.
+        data_model_parser = helpers.get_data_model_parser(data_model_name=data_model)
+
+        # Check the data model type
+        assert (data_model == 'example.model.csv') == (data_model_parser.model_type == 'CSV')
```

Comment:
Dictionary to look up that the value is correct instead of having two assert statements.

Reply:
Done!

Reply:
Yeah I agree, having just one assertion that checks the `DATA_MODEL_DICT` against the model parser model type would be easier to maintain; we wouldn't need to add new assertions each time we added a new supported format.